screens/Scan/ScanScreen.tsx

```diff
@@ -158,6 +163,16 @@ export const ScanScreen: React.FC = () => {
           progress
         />
       </Centered>
+      {!controller.isEmpty && (
```

Comment:
can you add this Overlay component inside a function and call the function inside loadQRScanner method, since we are following this way in scanScreen.tsx

Reply:
Refactored